common/test/acceptance/tests/lms/test_oauth2.py

```diff
@@ -56,5 +56,6 @@ def test_accepting_redirects(self):
         self.oauth_page.confirm()
 
         # This redirects to an invalid URI.
-        query = self._qs(self.browser.current_url)
+        self.oauth_page.wait_for_element_absence('input[name=authorize]', 'Authorization button is not present')
```

Comment:
Please move this line up so it's right under the .confirm() so it's clear the the test code pattern is:
* do something
* wait for synchronization
* 
* do the next thing
* etc